### PR TITLE
update list for transient activation

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

Added
- `Clients.openWindow()`
  - https://developer.mozilla.org/en-US/docs/Web/API/Clients/openWindow
  - https://w3c.github.io/ServiceWorker/#clients-openwindow
- `ContactsManager.select()`
  - https://developer.mozilla.org/en-US/docs/Web/API/ContactsManager/select
  - https://w3c.github.io/contact-picker/#contacts-manager-select
- `DocumentPictureInPicture.requestWindow()`
  - https://developer.mozilla.org/en-US/docs/Web/API/DocumentPictureInPicture/requestWindow
  - https://wicg.github.io/document-picture-in-picture/#dom-documentpictureinpicture-requestwindow
- `EyeDropper.open()`
  - https://developer.mozilla.org/en-US/docs/Web/API/EyeDropper/open
  - https://wicg.github.io/eyedropper-api/#dom-eyedropper-open
- `Serial.requestPort()`
  - https://developer.mozilla.org/en-US/docs/Web/API/Serial/requestPort
  - https://wicg.github.io/serial/#requestport-method
- `WindowClient.focus()`
  - https://developer.mozilla.org/en-US/docs/Web/API/WindowClient/focus
  - https://w3c.github.io/ServiceWorker/#client-focus

also adjust the order of list

### CODE REVIEW

File: files/en-us/web/security/user_activation/index.md [hunk 1]
```diff
@@ -28,34 +28,39 @@ If an activation has been triggered, the user agent differentiates between two t
 
 APIs that require transient activation (list is not exhaustive):
 
-- [`beforeunload` event](/en-US/docs/Web/API/Window/beforeunload_event)
+- {{domxref("Window/beforeunload_event", "beforeunload")}} event
+- {{domxref("Clients.openWindow()")}}
 - {{domxref("Clipboard.read()")}}
 - {{domxref("Clipboard.readText()")}}
 - {{domxref("Clipboard.writeText()")}}
+- {{domxref("ContactsManager.select()")}}
```
Comment (hamishwillee) [inline files/en-us/web/security/user_activation/index.md:36]: Can we update the linked page to indicate the transient requirement?

File: files/en-us/web/security/user_activation/index.md [hunk 2]
```diff
@@ -28,34 +28,39 @@ If an activation has been triggered, the user agent differentiates between two t
...
+- `MediaDevices.getViewportMedia()`
 - {{domxref("MediaDevices.selectAudioOutput()")}}
 - `MediaStreamTrack.sendCaptureAction()`
-- `MediaDevices.getViewportMedia()`
-- {{domxref("MediaDevices.getDisplayMedia()")}}
 - {{domxref("Navigator.share()")}}
 - {{domxref("PaymentRequest.show()")}}
 - {{domxref("PresentationRequest.start()")}}
 - {{domxref("RemotePlayback.prompt()")}}
+- {{domxref("Serial.requestPort()")}}
```
Comment (hamishwillee) [inline files/en-us/web/security/user_activation/index.md:56]: Can we make sure this page also has the requirement listed?

File: files/en-us/web/security/user_activation/index.md [hunk 3]
```diff
@@ -28,34 +28,39 @@ If an activation has been triggered, the user agent differentiates between two t
...
 - {{domxref("USB.requestDevice()")}}
-- {{domxref("Keyboard.lock()")}}
 - {{domxref("Window.open()")}}
+- {{domxref("Window.queryLocalFonts()")}}
 - {{domxref("Window.showOpenFilePicker()")}}
 - {{domxref("Window.showSaveFilePicker()")}}
 - {{domxref("Window.showDirectoryPicker()")}}
-- `Window.getScreenDetails()`
```
Comment (hamishwillee) [inline files/en-us/web/security/user_activation/index.md:57]: Are you _sure_ this should be removed? I haven't looked carefully on this but I see  "Initiate a multi-screen experience from a single transient user activation" (if you are sure this is right I'll look closer).

**Comment by hamishwillee** [comment]
This looks good. Can we update the linked items that are missing the requirement for transient activation (there appear to be a couple).

File: files/en-us/web/security/user_activation/index.md (hunk 3 above)
Comment (skyclouds2001) [reply files/en-us/web/security/user_activation/index.md:57]: Well, I recheck again and I think it is a requirement, though not mention in the method algorithm

**Comment by skyclouds2001** [comment]
> This looks good. Can we update the linked items that are missing the requirement for transient activation (there appear to be a couple).

Got it!

**Review by hamishwillee — APPROVED** [review]
Excellent, thanks for the fixes!

**Pull request merged**